src/app/tests/suites/certification/Test_TC_BRAC_1_1.yaml

```diff
@@ -40,3 +40,35 @@ tests:
       response:
           constraints:
               type: uint16
+
+    - label: "Read the global attribute: AttributeList"
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+
+    #issue #11053 disabled steps below Global attributes missing from YAML framework
+    - label: "Read the global attribute: EventList"
+      disabled: true
+      command: "readAttribute"
+      attribute: "EventList"
+      response:
+          constraints:
+              type: list
+
+    - label: "Read the global attribute: AcceptedCommandList"
+      disabled: true
+      command: "readAttribute"
+      attribute: "AcceptedCommandList"
+      response:
+          constraints:
+              type: list
+
+    - label: "Read the global attribute: GeneratedCommandList"
+      disabled: true
```

Comment:
This is supported too.

Reply:
i will remove disabled if it is added while rebasing